src/main/java/org/folio/dew/batch/JobCompletionNotificationListener.java

```diff
@@ -138,6 +139,8 @@ private void processJobUpdate(JobExecution jobExecution, boolean after) {
 
     kafka.send(KafkaService.Topic.JOB_UPDATE, jobExecutionUpdate.getId().toString(), jobExecutionUpdate);
     if (after) {
+      FolioExecutionScopeExecutionContextManager.endFolioExecutionContext();
+      log.info("FOLIO context closed.");
```

Comment:
The same as here https://github.com/folio-org/mod-data-export-worker/pull/254/files#r915819981. Just to avoid extra line with logger we can include it in the endFolioExecutionContext() body.

Reply:
It should be debug log level, not info.